mappings/net/minecraft/world/storage/StorageIoWorker.mapping

```diff
@@ -9,29 +11,45 @@ CLASS net/minecraft/unmapped/C_ffgsaknh net/minecraft/world/storage/StorageIoWor
 		ARG 2 dsync
 		ARG 3 executorName
 	METHOD close close ()V
+	METHOD m_bcdbqcmy (Lnet/minecraft/unmapped/C_ffgsaknh$C_bwprgfor;)Ljava/util/concurrent/CompletableFuture;
+		ARG 0 result
+	METHOD m_ehxqxhtz (Ljava/lang/Void;)Ljava/util/concurrent/CompletionStage;
+		ARG 1 v
+	METHOD m_fclorhxg (Lnet/minecraft/unmapped/C_hhlwcnih;Lnet/minecraft/unmapped/C_ynrszrtu;)Lnet/minecraft/unmapped/C_ffgsaknh$C_bwprgfor;
+		ARG 1 key
 	METHOD m_ipwbxzat setResult (Lnet/minecraft/unmapped/C_ynrszrtu;Lnet/minecraft/unmapped/C_hhlwcnih;)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 pos
 		ARG 2 nbt
 	METHOD m_jjukvdrp completeAll (Z)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 sync
+	METHOD m_jsyufhll (I)[Ljava/util/concurrent/CompletableFuture;
+		ARG 0 i
 	METHOD m_msmawqmw readChunkData (Lnet/minecraft/unmapped/C_ynrszrtu;)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 pos
 	METHOD m_nghssqns run (Ljava/util/function/Supplier;)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 task
-	METHOD m_nwcchuzp (II)Ljava/util/concurrent/CompletableFuture;
+	METHOD m_nwcchuzp getOrComputeBlendingStatus (II)Ljava/util/concurrent/CompletableFuture;
```

Comment:
```suggestion
	METHOD m_nwcchuzp getBlendingStatus (II)Ljava/util/concurrent/CompletableFuture;
```
Seems like this falls under this [convention](https://github.com/QuiltMC/quilt-mappings/blob/1.20-pre6/CONVENTIONS.md#getters-setters-withers-and-creators):
> Use "get" for non-boolean getters and other methods that calculate some property with no side effects other than caching a value in a private field.

Also `getOrComputeBlendingStatus` made me think it would act more like Java Map's `computeIfAbsent`, where a method to compute the value needs to be passed.